core/src/main/java/tc/oc/pgm/trigger/modifier/TriggerModifiers.java

```diff
@@ -0,0 +1,51 @@
+package tc.oc.pgm.trigger.modifier;
+
+import com.google.common.collect.ImmutableTable;
+import com.google.common.collect.Table;
+import java.util.function.Function;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.party.Party;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.trigger.Trigger;
+import tc.oc.pgm.util.xml.InvalidXMLException;
+
+public interface TriggerModifiers {
```

Comment:
Any specific reason you went with the current design instead of a design similar to how Filterables are done? (Decendants/Anscestors).
Maybe it even could be used as-is, with the outer/inner relation maybe just being saved in some other way like an Enum instead which maps to different usages of the Filterable API.

Reply:
I'm not exactly sure how the whole filterable API works, but an important point of triggerables is the ability to switch scopes and to be able to specifically know the scope something is running in. This will be especially relevant for variable-related triggers